ios/FluentUI.Demo/FluentUI.Demo/Demos/ColorAliasTokensDemoController.swift

```diff
@@ -0,0 +1,323 @@
+import UIKit
+
+class ColorAliasTokensDemoController: DemoTableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: TableViewCell.identifier)
+        tableView.backgroundColor = .lightGray
+    }
+
+    override func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int) {
+        if let headerView = view as? UITableViewHeaderFooterView {
+            headerView.textLabel?.textColor = .black
+        }
+    }
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return ColorAliasTokensDemoSection.allCases.count
+    }
+
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return ColorAliasTokensDemoSection.allCases[section].title
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return ColorAliasTokensDemoSection.allCases[section].rows.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: TableViewCell.identifier, for: indexPath)
+        let section = ColorAliasTokensDemoSection.allCases[indexPath.section]
+        let row = section.rows[indexPath.row]
+
+        cell.backgroundConfiguration?.backgroundColor = UIColor(dynamicColor: aliasTokens.colors[row])
+        cell.selectionStyle = .none
+
+        var contentConfiguration = cell.defaultContentConfiguration()
+        let text = "\(row.text)"
+        contentConfiguration.attributedText = NSAttributedString(string: text,
+                                                                 attributes: [
+                                                                    .foregroundColor: textColor(for: row)
+                                                                 ])
+        contentConfiguration.textProperties.alignment = .center
+        cell.contentConfiguration = contentConfiguration
+
+        return cell
+    }
+
+    private func textColor(for token: AliasTokens.ColorsTokens) -> UIColor {
+        let black = ColorValue(0x000000)
+        let white = ColorValue(0xFFFFFF)
+
+        switch token {
+        case .background1, .background1Pressed, .background1Selected, .background2, .background2Pressed, .background2Selected, .background3, .background3Pressed, .background3Selected, .background4, .background4Pressed, .background4Selected, .background5, .background5Pressed, .background5Selected, .background5BrandTint, .background6, .background6Pressed, .background6Selected, .backgroundDisabled, .brandBackground4, .background5BrandTintSelected, .brandBackgroundDisabled, .canvasBackground, .stencil1, .stencil2, .foregroundDisabled2, .foregroundOnColor, .brandForeground2, .stroke1, .stroke2, .strokeDisabled, .strokeFocus1, .foregroundDisabled1:
```

Comment:
Having this many cases on a single line will be very, very difficult to diff in the future. Can you put these either one per line, or at least a smaller number?